nameko/cli/run.py

```diff
@@ -1,14 +1,16 @@
 """Run a nameko service.
 
-Given a python path to a module containing a nameko service, will host and run
-it. By default this assumes a service class named ``Service``, but this can be
-provided via ``nameko run module:ServiceClass``.
+Given a python path to a module containing one or more nameko services, will
+host and run them. By default this will try to find "service-looking" classes
```

Comment:
By default it will find classes that look like services (anything with nameko entrypoints), ...